docs/production/management-commands.md

```diff
@@ -125,6 +125,7 @@ There are dozens of useful management commands under
 - `./manage.py export_single_user`: does a limited version of the [main
   export tools](export-and-import.md) containing just
   the messages accessible by a single user.
+- `./manage.py reactivate_stream`: Reactivates a stream.
```

Comment:
"Reactivates an archived stream." Let's link the word "reactivate" to the `/help/` page, ideally to a section heading for reactivating archived streams.